src/app/_blocks/EpisodeHead/index.tsx

```diff
@@ -0,0 +1,57 @@
+"use client";
+
+import React, { useEffect, useState } from "react";
+import BackButton from "../../_components/BackButton";
+import ContentTypePill from "@/app/_components/ContentTypePill";
+import { PodcastEpisode } from "@/payload/payload-types";
+import { formatDateTime } from "@/app/_utilities/formatDateTime";
+import AudioPlayer from "@/app/_components/AudioPlayer";
+import EpisodeFeaturedImage from "@/app/_components/EpisodeFeaturedImage";
+
+import { getEpisodeDuration } from "@/app/_utilities/getEpisodeDuration";
+import { getAudio } from "@/app/_utilities/getAudio";
+import { getImage } from "@/app/_utilities/getImage";
+
+
+
+export const EpisodeHead: React.FC<{ episode: PodcastEpisode }> = ({ episode }) => {
+
+  const {
+    title,
+    episodeFile,
+    publishedAt,
+    featuredImage,
+  } = episode;
+
+  // TODO: convert into conditional logic based on ContentType
+
+  // Initial undefined state
+  const {
+    audioFileSource,
+    audioFileType,
+  } = getAudio(episodeFile);
+
+
+  return (
+    <div style={{ display: "flex", backgroundColor: "#773BFF", position: "relative", padding: "20px" }}>
+      {/* First Column */}
+      <div style={{ flex: 1, marginRight: "20px" }}>
+        <div><BackButton /></div>
+        <div>
+          <ContentTypePill />
+          <h5>{title}</h5>
+          <h6>episode.podcastName property</h6>
```

Comment:
Is this meant to be <h6>{episode.podcastName}</h6>?

Reply:
based on Figma, yes. 


i've added an `h6` for now for refference but it will be formatted accordingly when we move on to styling